SVG.flood/programs/animations.flp

```diff
@@ -94,6 +114,14 @@ func animation(property, layer, timeline, hrefID, attributeType)
     end
 end
 
+func hrefValuesFrom(track, layer)
+    var values = []
+    for keyvalue in track.keyvaluesWithSimulatedSteps
+        values = values.appending(hrefFrom(keyvalue: keyvalue))
+    end
+    return values
+end
```

Comment:
do we need this function?

Reply:
I think we can just update the`svgValue` function instead, and just handle fill Images there no?

Reply:
remove this